frontend/server/src/main/java/org/pytorch/serve/util/PriorityLinkedBlockingDeque.java

```diff
@@ -98,9 +81,9 @@ private T unlinkFirst() {
     }
 
     public boolean isEmpty() {
+        // return true iff all deques are empty
```

Comment:
Maybe

```java
public boolean isEmpty() {
        // return true iff all deques are empty
        return this.priorityDeques.reduceValues(Long.MAX_VALUE, LinkedBlockingDeque::isEmpty, Boolean::logicalAnd);
    }
```
would be a bit shorter? Github does not allow me to add this as a suggestion.